noir_stdlib/src/field/mod.nr

```diff
@@ -132,6 +165,30 @@ impl Field {
     }
 }
 
+/// Convert a big endian byte array to a field element by modding
+pub fn from_le_bytes<let N: u32>(bytes: [u8; N]) -> Field {
+    let mut v = 1;
+    let mut result = 0;
+
+    for i in 0..N {
+        result += (bytes[i] as Field) * v;
+        v = v * 256;
+    }
+    result
+}
+
+/// Convert a little endian byte array to a field element by modding
+pub fn from_be_bytes<let N: u32>(bytes: [u8; N]) -> Field {
```

Comment:
These should be class methods on Field rather than independent functions.

Reply:
Done